robot_X: 0.34
robot_Y: 0.34
robot_THETA: 0.0
